config/metadata/rulesMeta.yaml

```diff
@@ -651,6 +651,12 @@ groups:
         summary: is the comparison operator to use.
         description: >
           The comparison operator to use. String comparisons are case-sensitive.
+          For most cases, use the negative operators (`!=`, `does-not-contain`, and
+          `not-exists`) in a rule with a scope of "span". WARNING: A negative operator
+          in a rule with a scope of "trace" (the default) can cause the rule to match
+          a trace if *any* span in the trace matches the negative condition, for example
+          a single span in a trace lacks the specified field (`not-exists`).
+          This is rarely the matching behavior you want.
```

Comment:
"This" somewhat obscures the intended outcome, and I understand the example but it seems to be an incomplete thought? (Or maybe more clarification is needed.)

Reply:
I think an example is counterproductive unless you want to get much more detailed. Maybe breaking it up this way helps?

> WARNING: Rules can have `Scope: trace` or `Scope: span`; a negative operator with `Scope: trace` will be true if **any** single span in the entire trace matches the negative condition.

Reply:
Oooooh, I like this suggested phrasing a lot more. And I agree about omitting the example. Thank you @kentquirk! cc: @robbkidd

Reply:
Revised to this wording in 893d2bc